tachyon/zk/plonk/vanishing/vanishing_utils.h

```diff
@@ -21,11 +21,12 @@ namespace tachyon::zk::plonk {
 // Calculate ζ = g^((2ˢ * T) / 3).
 template <typename F>
 constexpr F GetZeta() {
-  CHECK_EQ(F::Config::kTrace % math::BigInt<F::kLimbNums>(3),
-           math::BigInt<F::kLimbNums>(0));
-  return F::FromMontgomery(F::Config::kSubgroupGenerator)
-      .Pow(F(2).Pow(F::Config::kTwoAdicity).ToBigInt() * F::Config::kTrace /
-           math::BigInt<F::kLimbNums>(3));
+  using BigInt = typename F::BigIntTy;
+  CHECK_EQ(F::Config::kTrace % BigInt(3), BigInt(0));
+  BigInt exp = F::Config::kTrace;
```

Comment:
How about adding a comment like the following
// The result of exponential does not exceed modular of scalar field.